components/array-object-table/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "@govuk-frederic/array-object-table",
+  "version": "0.0.1",
+  "dependencies": {
+    "@govuk-frederic/table": "^0.0.3",
+    "@govuk-frederic/utils": "^0.0.1"
+  },
+  "devDependencies": {
+    "@govuk-react/storybook-components": "^0.2.4",
+    "@storybook/addon-knobs": "^3.4.2"
+  },
+  "peerDependencies": {
+    "emotion": ">=9",
+    "prop-types": ">=15",
+    "react": ">=16.2.0",
+    "react-emotion": ">=9"
+  },
+  "scripts": {
+    "build": "npm run build:lib && npm run build:es",
+    "build:lib": "rimraf lib && babel src -d lib --source-maps",
+    "build:es": "rimraf es && cross-env BABEL_ENV=es babel src -d es --source-maps",
+    "docs": "doc-component ./lib/index.js ./README.md"
+  },
+  "main": "lib/index.js",
+  "module": "es/index.js",
+  "author": "Gavin Orland",
+  "license": "MIT",
+  "homepage": "https://github.com/stevesims/govuk-frederic#readme",
+  "description": "govuk-frederic: Frederic project specific components.",
```

Comment:
Should the description for this package file not be about what `ArrayObjectTable` is and does?

Reply:
Yes, ideally - this is something that needs doing for all Fred components (and `govuk-react` ones!). Added https://github.com/stevesims/govuk-frederic/issues/54.